server/app/serializers/fusor/deployment_serializer.rb

```diff
@@ -9,8 +9,6 @@ class DeploymentSerializer < ActiveModel::Serializer
                :rhev_storage_type, :rhev_storage_address, :rhev_cpu_type, :rhev_share_path,
                :rhev_export_domain_name, :rhev_export_domain_address,
                :rhev_export_domain_path, :rhev_local_storage_path,
-               :rhev_gluster_node_name, :rhev_gluster_node_address,
```

Comment:
Do we want to add these back into the serializer?

  rhev_gluster_node_name
  rhev_gluster_node_address
  rhev_gluster_ssh_port